src/app/dataset-view/additional-components/dataset-settings-component/tabs/scheduling/ingest-trigger-form/ingest-trigger-form.component.ts

```diff
@@ -23,7 +23,7 @@ import { DatasetSchedulingService } from "../../../services/dataset-scheduling.s
 })
 export class IngestTriggerFormComponent extends BaseComponent implements OnInit {
```

Comment:
Note: potentially we have a design issue here. Period/CRON stuff is not limited to Ingest flow, I think we will have other cases, like Soft Compacting and GC very soon. There is a sense to isolate this part in a sub-component, which you would reuse for other flow types in future.

Not insisting on doing this right now, we can have it done later, as soon as we need the 2nd use.

Reply:
We will definitely do it, but later